### PR TITLE
R4 Immunization Documentation added for Patient Access

### PR DESCRIPTION
**Pull request by ShrutiNHegde**
Description
----
Adding R4 Immunization Documentation for Patient Access.

Screenshot before merge
-----
heading-afterchanges
Overiew-after changing
Retrive by id-after changing
Search-After changing
Patient Autherization

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by aki-mathur — APPROVED** [review]

File: content/millennium/r4/medications/immunization.md [hunk 1]
```diff
@@ -81,6 +81,19 @@ Notes:
 <%= headers status: 200 %>
 <%= json(:R4_IMMUNIZATION_BUNDLE) %>
 
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization?patient=12724066
```
Comment (ashu-bdvt) [inline content/millennium/r4/medications/immunization.md:88]: Update url to https://fhir-ehr.cerner.com

File: content/millennium/r4/medications/immunization.md [hunk 2]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
 <%= headers status: 200 %>
 <%= json(:R4_IMMUNIZATION_ENTRY) %>
 
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Completed Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M197197986
```
Comment (ashu-bdvt) [inline content/millennium/r4/medications/immunization.md:130]: Update url to https://fhir-ehr.cerner.com

File: content/millennium/r4/medications/immunization.md [hunk 3]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
...
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_PATIENT_ENTRY) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Entered in Error Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M211860902
```
Comment (ashu-bdvt) [inline content/millennium/r4/medications/immunization.md:141]: Update url to https://fhir-ehr.cerner.com

File: content/millennium/r4/medications/immunization.md [hunk 4]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
...
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_ENTERED_IN_ERROR_STATUS) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Not done Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/HM178147794
```
Comment (ashu-bdvt) [inline content/millennium/r4/medications/immunization.md:152]: Update url to https://fhir-ehr.cerner.com

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Add screenshots of the changes done (patient authorization type added) as screenshots by running the code locally

**Comment by ShrutiNHegde** [comment]
Screenshots after changes: running locally
heading-afterchanges
Overview-afterchanges
PatientAutherizationRequest-completed-read
PatientAutherizationRequest-entered in error-read
PatientAutherizationRequest-not done-read
PatientAutherizationRequest-search

File: content/millennium/r4/medications/immunization.md (hunk 1 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Addressed in commit ID df37656

File: content/millennium/r4/medications/immunization.md (hunk 2 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Addressed in commit ID df37656

File: content/millennium/r4/medications/immunization.md (hunk 3 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Addressed in commit ID df37656

File: content/millennium/r4/medications/immunization.md (hunk 4 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Addressed in commit ID df37656

**Review by ashu-bdvt — APPROVED** [review]

File: content/millennium/r4/medications/immunization.md [hunk 5]
```diff
@@ -81,6 +81,19 @@ Notes:
...
+#### Patient Authorization Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization?patient=12724066
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_PATIENT_BUNDLE) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
```
Comment (aki-mathur) [inline content/millennium/r4/medications/immunization.md:95]: in place of Note just add the `<%= disclaimer %>` tag

File: content/millennium/r4/medications/immunization.md [hunk 6]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
...
+#### Patient Authorization Request For Completed Status
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M197197986
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_PATIENT_ENTRY) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
```
Comment (aki-mathur) [inline content/millennium/r4/medications/immunization.md:137]: In place of note add `<%= disclaimer %>`

File: content/millennium/r4/medications/immunization.md [hunk 7]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
...
+#### Patient Authorization Request For Entered in Error Status
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M211860902
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_ENTERED_IN_ERROR_STATUS) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
```
Comment (aki-mathur) [inline content/millennium/r4/medications/immunization.md:148]: In place of note add `<%= disclaimer %>`

File: content/millennium/r4/medications/immunization.md [hunk 8]
```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
...
+#### Patient Authorization Request For Not done Status
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/HM178147794
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_NOT_DONE_STATUS) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
```
Comment (aki-mathur) [inline content/millennium/r4/medications/immunization.md:159]: In place of note add `<%= disclaimer %>`

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: content/millennium/r4/medications/immunization.md (hunk 5 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Fixed in commit 78772b8

File: content/millennium/r4/medications/immunization.md (hunk 6 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Fixed in commit 78772b8

File: content/millennium/r4/medications/immunization.md (hunk 7 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Fixed in commit 78772b8

File: content/millennium/r4/medications/immunization.md (hunk 8 above)
Comment (ShrutiNHegde) [reply content/millennium/r4/medications/immunization.md]: Fixed in commit 78772b8

**Review by aki-mathur — APPROVED** [review]

**Comment by VelocityKnight** [comment]
Post merge verification:
Screen Shot 2020-12-10 at 6 19 52 PM
Screen Shot 2020-12-10 at 6 19 29 PM
Screen Shot 2020-12-10 at 6 19 11 PM
Screen Shot 2020-12-10 at 6 18 31 PM
Screen Shot 2020-12-10 at 6 18 22 PM

**Pull request merged**